tools/update-versions.js

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env node
+'use strict';
+
+const fs = require('fs-extra');
+const path = require('path');
+const root = path.join(__dirname, '..');
+
+const manifest = require(`${root}/manifest.json`);
+const pkg = require(`${root}/package.json`);
+
+const good = '\x1b[32m%s\x1b[0m';
+const warn = '\x1b[36m%s\x1b[0m';
+
+function exit(err) {
+  if (err) {
+    console.error(err);
+  }
+  process.exit(err ? 1 : 0);
+}
+
+function verToArray(v) {
+  return v.replace('v', '').split('.').map(Number);
+}
+
+// Simple compare function since we can't require semverCompare here
+function compare(v1, v2) {
+  if (v1 === v2) {
+    return 0;
+  }
+  const [maj1, min1, pat1] = verToArray(v1);
+  const [maj2, min2, pat2] = verToArray(v2);
+  const majMatch = maj1 === maj2;
+  const minMatch = min1 === min2;
+  if (
+    maj1 > maj2 ||
+    majMatch && min1 > min2 ||
+    majMatch && minMatch && pat1 > pat2
+  ) {
+    return 1;
+  }
+  return -1;
+}
+
+async function updateVersions() {
+  const result = compare(manifest.version, pkg.version);
+  let file, obj;
+  if (result === 0) {
+    return console.log(good, 'Manifest & package versions match');
+  } else if (result > 0) {
+    pkg.version = manifest.version;
+    file = 'package.json';
+    obj = pkg;
+  } else {
+    manifest.version = pkg.version;
+    file = 'manifest.json';
+    obj = manifest;
+  }
+  console.log(warn, `Updating ${file} to ${pkg.version}`);
+  return fs.writeFile(`${root}/${file}`, JSON.stringify(obj, null, '  ') + '\n');
```

Comment:
Will it produce a mixed-linefeed file if git is used with autocrlf checkout on Windows OS? If so, you need to detect the linefeed type and use it.

Reply:
Hmm, good question... what are your thoughts on adding a `.gitattributes` file with a `* text=auto` entry?

Reply:
I didn't use it so idk how it'll help in case the file was CRLF but your script added only LF.
I certainly don't want to enforce any particular kind of linefeed.

Reply:
Supposedly that setting normalizes the CRLF for your OS when you check-in and restores them when you check-out - ref: https://git-scm.com/docs/gitattributes#_effects

Reply:
My question still stands.
Can you check what happens?

Reply:
IMO this setting won't help in case of mixed linefeeds.

Reply:
Ok, now I just need to figure out how to detect the line feeds... I guess I'd need to read in a file.

Reply:
There is no need to regenerate the entire JSON file. Try replacing the old version with the new one so other stuff (line feed, indent, etc) stay untouched. Something like:
```js
const match = matchVersion(manifestContent)
const newManifestContent = manifestContent.slice(0, match.index) + newVersion + manifestContent.slice(match.index + match[0].length)
```